frontend/controller/service-worker.js

```diff
@@ -313,8 +260,16 @@ sbp('sbp/selectors/register', {
             }),
             primaryButton: L('Close')
           })
+          // NOTE: we do not need to setup an ONLINE event listener here because one is already setup in main.js
+          throw e // give up
         }
-        throw e
+        // this outer promise is a way to wait on this sub-call to finish without getting the eventQueue stuck
+        retryAttemptsPromise = new Promise((resolve, reject) => {
+          // try again in 1 second
+          setTimeout(() => {
+            sbp('service-worker/setup-push-subscription', attemptNumber + 1).then(resolve).catch(reject)
+          }, 12e3)
```

Comment:
The comment says 1 second yet the wait is 12 seconds.

Reply:
Yes, good catch. And also, I think this is supposed to be 5 seconds.